.kitchen.yml

```diff
@@ -1,62 +1,77 @@
 ---
 driver:
-  name: vagrant
+  name: docker
```

Comment:
Take a look at https://github.com/dev-sec/chef-mysql-hardening/blob/master/.kitchen.yml
tl;dr dokken abstracts away from of the docker stuff that you don't care about in a CI environment (i.e. the `run_opts` and `init`options you have in the travis config)

Your config looks like it works fine the way it is, so just a thought if you have time or for the future :)